lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -256,7 +262,7 @@ <h3 class="hd hd-3">${_("Coupon Code List")}</h3>
          }
          if($('#invoice_number').val() == "") {
                      $('#error-msg').attr('class','error-msgs')
-                     $('#error-msg').html("${_('The Invoice Number field cannot be empty.') | n, h, js_escaped_string}").show();
+                    $('#error-msg').text("${_('The Invoice Number field cannot be empty.') | n, js_escaped_string}").show();
```

Comment:
@robrap I'm struggling to understand whether or not js_escaped_string is necessary here. Can you take a look and enlighten me?